server/templates/pages/src/index.html

```diff
@@ -32,7 +32,7 @@
           <!-- endbuild -->
         <![endif]-->
     </head>
-    <body>
+    <body{{#if flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/if}}>
```

Comment:
> I believe this should be:
> ```
> {{#flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/flowId}}
> ```

Really? Everywhere else in this template uses `#if`, the handlebars docs say to do it that way and when I try your suggestion locally, it doesn't work. What am I missing?